### ISSUE TITLE
Doesn't work in IE11 because of proxy

### ISSUE THREAD

**Issue by b-strauss** [AmazingDreams/vue-matomo#17]
Doesn't work in IE11 because of proxy

**Comment by AmazingDreams** [comment]
Hmm, could've known. Do you have any suggestions?

**Comment by b-strauss** [comment]
Maybe remove the Proxy and use `_paq.push` directly? 😄

Proxy is pretty hard to polyfill.

**Comment by b-strauss** [comment]
Hmm the proxied object is now part of the public API.

```js
Vue.prototype.$piwik = Matomo
Vue.prototype.$matomo = Matomo
```

I guess that would be a breaking change.

**Comment by AmazingDreams** [comment]
Yeah, the issue is I need the proxy to circumvent the time it takes to load the Matomo script. Making the api immediately usable.

I think the best way forward is check for browser support on the proxy, and otherwise delay setting the matomo client to when the script is loaded.

**Comment by snaill** [comment]
Doesn't work in Safari 9.
Can you do nothing in ES5 browser ?
First check Proxy object like this:
```
if ((typeof window.Proxy) == undefined) {
    console.log('browser not support Proxy.');
}
```

**Comment by AmazingDreams** [comment]
The only real solution is to include the Matomo JS source with the build unfortunately. Either there's a delay in loading which confuses users, or there's no proxy support.

I'll see what I can do
